test/Microsoft.IdentityModel.Tests/ClaimSets.cs

```diff
@@ -156,6 +174,12 @@ public static List<Claim> DefaultClaims
             private set;
         }
 
+        public static Dictionary<DefaultClaimType, string> DefaultClaimsMapping
```

Comment:
Even though it is a bit slower, I have found it better to return a new object each time so that each test variant is free to tweak the test artifact as they choose.